source/index.txt

```diff
@@ -1 +1,55 @@
-Test Build
+.. _visual-studio-extension:
+
+.. _visual-studio-extension-landing:
+
+===============================
+{+product+}
+===============================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   Installation </install>
+   Builders </builders>
+   LINQ </linq>
+   Configuration </configuration>
+   FAQ </faq>
+   What's New </whats-new>
+
+Overview
+--------
+
+The `{+product+} <{+product-home-page+}>`__ is a NuGet package
+that helps you understand the following about your {+driver-long+} 
+code:
+
+- How your code translates into the {+query-api+}
+- If your code includes unsupported LINQ or builder expressions
+
+The {+product+} can analyze the following groups of {+driver-short+} expressions:
+
+- `Builders <{+driver-docs+}reference/driver/definitions/>`__
+- `LINQ <{+driver-docs+}reference/driver/crud/linq/>`__
+- `LINQ3 <{+driver-docs+}reference/driver/crud/linq3/>`__
+
+Read the following sections of this guide to learn how to install the
+{+product+}, how to use its features, and how to configure it:
+
+- :doc:`Install the extension </install>`
+- :doc:`Analyze your builder expressions </builders>`
+- :doc:`Analyze your LINQ expressions </linq>`
+- :doc:`Configure the extension </configuration>`
+
+To view answers to frequently asked questions, see the :doc:`FAQ </faq>` page.
+
+To view the differences between versions of the extension,
+see the :doc:`What's New <whats-new>` page. 
+
+.. note:: Roslyn Analyzer
```

Comment:
Nice, I thought to add this to FAQ, but "note" actually looks better.

Reply:
Thanks! And we can do both if you think it makes sense to add to the FAQ as well. I'll link the preceding comment to the [FAQ page JIRA ticket](https://jira.mongodb.org/browse/DOCSP-19710) so we remember to consider Roslyn Analyzers when writing the FAQ page.